src/forms/settings-dialog.cpp

```diff
@@ -807,7 +809,8 @@ void PluginWindow::add_row_from_hook(const MidiHook *hook) const
 	auto *item_item = new QTableWidgetItem(hook->item);
 	auto *audio_item = new QTableWidgetItem(hook->audio_source);
 	auto *media_item = new QTableWidgetItem(hook->media_source);
-	auto *hotkey_item = new QTableWidgetItem(Utils::get_hotkey_value(hook->hotkey));
+	obs_hotkey_t *hotkey = hook->getHotkey();
+	auto *hotkey_item = hotkey ? new QTableWidgetItem(obs_hotkey_get_description(hotkey)) : new QTableWidgetItem();
```

Comment:
add the hotkey model to this instead of the text.

Reply:
Hmmm... I don't think this would work... At least as I understood models

Reply:
As long we use `QTableWidget` it is not possible to use models. For models we need `QTableView`
https://doc.qt.io/qt-5/modelview.html#1-3-overview-of-the-model-view-widgets

Using `QTableView` would implicate to change actually how the whole table is rendered. It is for me too much right now...

`storeData()` didn't work neither... Xcode was claiming about passing pointer of hotkey.